packages/permissionless/accounts/coinbase/constants.ts

```diff
@@ -0,0 +1,8 @@
+export const COINBASE_SMART_WALLET_IMPLEMENTATION_ADDRESS =
+    "0x000100abaad02f1cfC8Bbe32bD5a564817339E72"
+
+export const COINBASE_SMART_WALLET_FACTORY_ADDRESS =
+    "0x0BA5ED0c6AA8c49038F819E587E2633c4A9F428a"
+
+export const ERC1271InputGeneratorByteCode =
```

Comment:
we've actually moved away from using this in favor of just manually computing the replaySafeHash. Will try to share code

Reply:
yeah would love to see how you guys do that, thanks

Reply:
Here you go:

```ts
import { hashTypedData, type Address, type Hash } from 'viem'

function toReplaySafeHash({
  address,
  chainId,
  hash,
}: { address: Address; chainId: number; hash: Hash }) {
  return hashTypedData({
    domain: {
      chainId,
      name: 'Coinbase Smart Wallet',
      verifyingContract: address,
      version: '1',
    },
    types: {
      CoinbaseSmartWalletMessage: [
        {
          name: 'hash',
          type: 'bytes32',
        },
      ],
    },
    primaryType: 'CoinbaseSmartWalletMessage',
    message: {
      hash,
    },
  })
}
```

Reply:
thanks, works.